UnityProject/Assets/Scripts/HealthV2/Living/Mutations/Brain/AlcoholEffectMutation.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEngine;
+
+namespace HealthV2.Living.Mutations.Brain
+{
+	[CreateAssetMenu(fileName = "AlcoholEffectMutation", menuName = "ScriptableObjects/Mutations/AlcoholEffectMutation")]
+	public class AlcoholEffectMutation  : MutationSO
```

Comment:
I agree ethimologically it doesn't make sense, but the player effect system didn't take off. Seems like we are making everything a mutation now, which isn't really that bad.

Reply:
Wouldn't it be better to make it a status effect then, rather than a mutation? Because drunkness isn't a mutation, it's an effect caused by something you can find in multiple factors that also isn't just alcohol.

Reply:
this is not the drunkenness itself, It's just modifying Susceptibility to drunkenness